code/modules/hydroponics/seed_datums/flowers.dm

```diff
@@ -23,7 +23,7 @@
 	name = "poppies"
 	seed_name = "poppy"
 	display_name = "poppies"
-	chems = list(/singleton/reagent/nutriment = list(1,20), /singleton/reagent/bicaridine = list(1,10))
+	chems = list(/singleton/reagent/nutriment = list(1,20), /singleton/reagent/morphine = list(1,10))
```

Comment:
i don't think that the poppies that are available on the ship should be opioid poppies but rather neutered poppies.

Reply:
Medical requires this for morphine and oxycomorphine now.

Reply:
they can make tramarine and artificial morphine.

Reply:
This will be available, as Biesel does allow the growing of poppies upon consultation with lore.